include/boost/http_proto/detail/impl/workspace.hpp

```diff
@@ -7,9 +7,12 @@
 // Official repository: https://github.com/cppalliance/http_proto
 //
 
+#pragma once
+
 #ifndef BOOST_HTTP_PROTO_DETAIL_IMPL_WORKSPACE_HPP
 #define BOOST_HTTP_PROTO_DETAIL_IMPL_WORKSPACE_HPP
 
+#include <boost/http_proto/detail/workspace.hpp>
```

Comment:
Hmm this isn't right. detail/impl/workspace.hpp is not designed to be included on its own, it is included at the bottom of the corresponding declaration file at detail/workspace.hpp. We should not be including the declaration file here. What's going on?